server/auth/store.go

```diff
@@ -968,6 +968,8 @@ func NewAuthStore(lg *zap.Logger, be AuthBackend, tp TokenProvider, bcryptCost i
 	tx.Unlock()
 	be.ForceCommit()
 
+	as.refreshRangePermCache(tx)
```

Comment:
Please move this inside the transaction lock

Reply:
done, thanks.